### ISSUE TITLE
guest de-spawn stops working after resize

### ISSUE THREAD

**Issue by cheweytoo** [fidwell/OpenRct2-MapResizer#1]
If the edge with the guest spawn point is moved, guest spawning still works fine (from the original tile's edge, which is now within the map), but guest de-spawning stops working. Guests leaving the park recoil from the (de-)spawn edge and have nowhere to go:

![no_despawn](https://github.com/fidwell/OpenRct2-MapResizer/assets/29800630/9ac0f6a9-6ae5-4abb-84b1-f6028ead9c38)

This might not be an issue with the plugin, but with the game itself though…

**Comment by fidwell** [comment]
Yeah I think this is more of a game thing than a plugin thing. If you extend your path all the way to the edge of the map, those peeps should be able to despawn as usual.

**Comment by cheweytoo** [comment]
They don't. Building the path to the map edge doesn't help: I have to manually delete and recreate the spawn point.

Should I file a bug with OpenRCT2?

**Comment by fidwell** [comment]
Huh, that's odd. I loaded Forest Frontiers, extended the bottom edge where the spawner was, and extended the path to the edge, and peeps were able to leave just fine. Maybe it's because it's over water? Doesn't seem likely. But something seems wrong there either way. If you can consistently get it to behave like that, then yeah I'd file an issue on the OpenRCT2 GitHub.
